nimble/host/include/host/ble_eddystone.h

```diff
@@ -106,6 +106,29 @@ int ble_eddystone_set_adv_data_url(struct ble_hs_adv_fields *adv_fields,
                                    uint8_t url_body_len, uint8_t suffix,
                                    int8_t measured_power);
 
+/**
+ * Configures the device to advertise Eddystone TLM (Unencrypted) beacons.
+ *
+ * @param adv_fields            The base advertisement fields to transform into
+ *                                  an eddystone beacon.  All configured fields
+ *                                  are preserved; you probably want to clear
+ *                                  this struct before calling this function.
+ * @param battery_voltage       The 2-byte battery voltage in mV to advertise.
+ * @param temperature           The 2-byte temperature to advertise.
+ * @param adv_pdu_count         The 4-byte total packet count to advertise.
+ * @param time_since_power_up   The 4-byte count in 0.1 Seconds since the device 
+ *                                  started / rebooted.
+ *
+ * @return                      0 on success;
+ *                              BLE_HS_EBUSY if advertising is in progress;
```

Comment:
I don't think this will ever return `BLE_HS_EBUSY` since it's perfectly fine to change advertising data while advertising is in progress

Reply:
I assumed it is possible that the User stops advertising and advertises TLM data, hence the similarity in returns.